packages/button-theme/src/styles.tsx

```diff
@@ -0,0 +1,292 @@
+            background-color: #000;
+            text-decoration: underline;
+          }
+
+          &:focus {
+            box-shadow: 0px 0px 0px 2px #FBDD01 inset !important;
+          }
+        `,
+    },
+    standard: {
+      input: `
+            border: 1px solid #000;
+          `,
+      container: `
+          border: 1px solid #707070;
+          background-color: #F2F2F2;
+        `,
+      header: `
+          color: #000;
+        `,
+      content: `
+          color: #000;
+        `,
+      close: `
+          color: #000;
+          border: 1px solid #000;
+
+          &:hover {
+            background: #000;
+            color: #fff;
+          }
+        `,
+      group: `
+          color: #000;
+        `,
+    },
+    danger: {
+      container: `
+            border: 2px solid #DB2828;
+            background-color: #FEE8E6;
+          `,
+      header: `
+            color: #db2828;
+          `,
+      content: `
+            color: #db2828;
+          `,
+      close: `
+            color: #db2828;
+            border: 1px solid #db2828;
+
+            &:hover {
+              background: #db2828;
+              color: #fff;
+            }
+          `,
+      group: `
+            color: #db2828;
+          `,
+    },
+    success: {
+      container: `
+            border: 2px solid #1ebc30;
+            background-color: #E5F9E6;
+          `,
+      header: `
+            color: #1ebc30;
+          `,
+      content: `
+            color: #1ebc30;
+          `,
+      close: `
+            color: #1ebc30;
+            border: 1px solid #1ebc30;
+
+            &:hover {
+              background: #1ebc30;
+              color: #fff;
+            }
+          `,
+      group: `
+            color: #1ebc30;
+          `,
+    },
+  },
+  disabled: `
+      font-size: 1rem;
+      color: #fff !important;
+      background-color: #2476ED !important;
+      box-shadow: 0px 0px 0px 1px #2476ED inset !important;
+      text-decoration: none !important;
+      opacity: 0.43;
+    `,
+  flex: {
+    container: `
+        display: flex;
+      `,
+    group: `
+        margin: auto 0.5rem;
+      `,
+  },
+  resize: {
+    none: { input: 'resize: none;' },
+    both: { input: 'resize: both;' },
+    horizontal: { input: 'resize: horizontal;' },
+    vertical: { input: 'resize: vertical;' },
+  },
+  required: {
+    label: `
+            &:after {
+              margin: -0.2em 0em 0em 0.2em;
+              content: '*';
+              color: #DB2828;
+            }
+          `,
+  },
+  size: {
+    small: {
+      button: `
+              font-size: 0.8rem;
+            `,
+      container: `
+              font-size: 0.8rem;
+            `,
+
+      label: `
+            font-size: 0.8rem;
+          `,
+
+      input: `
+          font-size: 0.8rem;
+        `,
+      header: `
+              font-size: 1rem;
+            `,
+      content: `
+              font-size: 0.8rem;
+              padding: 0.4rem;
```

Comment:
Added padding to content to ensure the close button does not extend beyond the container